packages/browser-integration-tests/jest.config.js

```diff
@@ -1,3 +1,3 @@
 const { createJestTSConfig } = require('@internal/config')
 
-module.exports = createJestTSConfig()
+module.exports = createJestTSConfig(__dirname)
```

Comment:
Adding nested packages (i.e. packages/consent/*) broke the previous jest work I did on allowing us to run `jest --watch` for the entire monorepo in a single jest instance without having to compile tsc after every change. The config is much more resillient now when it comes to project structure.